### PR TITLE
Restore and adjust player event "denied"

### PR DESCRIPTION
**Pull request by kai-li-wop**
The player event "denied" is deactivated in World of Padman and the code has been partially removed. Denied is awarded as an anti-reward, for example, when a player has not won a race for a powerup and it has been snatched from under his nose. The Impressive reward is misused in World of Padman and instead of a medal with announcement only the gleeful laughter of the player sounds hehe1/hehe2. This approach can also be used for the denied event. If a player doesn't get a powerup any more because another player snatched it away, the laughter of the other player should be heard. In this respect, no new announcer sound is needed here.

ToDo "denied"
- [x] Check for possible "16bits limit" issue (also connected to #15) --> no issue here!
- [x] ~~Restore the code for "denied"~~
- [x] Use hehe2 sounds for "denied", but of opposite/other playerwho picks up the item instead

Implementation:
Instead of restoring the vQ3 code to play a global "denied" sound, the person who picks up a powerup, such as PadPower, now laughs at other opponents who wanted to pick up the same powerup but weren't fast enough. This happens when there is at least one enemy player within 192 units range of the item who is facing the item and can see it directly. If this applies to more than one opponent, only one laugh is made. It works only with powerups, not holdables or anything else. Also does not apply to teammates.

### CODE REVIEW

File: code/game/g_items.c [hunk 1]
```diff
@@ -453,7 +464,7 @@ void Touch_Item(gentity_t *ent, gentity_t *other, trace_t *trace) {
 		break;
 	case IT_POWERUP:
 		respawn = Pickup_Powerup(ent, other);
-		predict = qfalse;
+		//		predict = qfalse;
```
Comment (mgerhardy) [inline code/game/g_items.c:467]: note for myself: check what the impact is on this change

Comment (kai-li-wop) [reply code/game/g_items.c:467]: I also don't know the full impact of this change, but without it I couldn't hear the denied award sound. Could be an issue on my side only.

Comment (mgerhardy) [reply code/game/g_items.c:467]: safe to disable it

**Comment by 3aTmE** [comment]
Tried to trigger the player event "denied" with an local game with 2 clients on `wop_anteroom`.

Worked like it should! 

I also tried different Characters. 

The two I've tested so far worked like intended (Padman / Fatpad )

**Pull request merged**